src/web/app/pipes/pipes.module.spec.ts

```diff
@@ -0,0 +1,57 @@
+import { Pipes } from './pipes.module';
+import { ResponseStatusPipe } from './sessionResponseStatus.pipe';
```

Comment:
Angular's file names should follow `kebab-case` convention, so this should be `session-response-status.pipe.ts`.